mono/mini/mini-exceptions.c

```diff
@@ -3195,6 +3213,7 @@ throw_exception (MonoObject *ex, gboolean rethrow)
 		for (l = ips; l; l = l->next) {
 			trace = g_list_append (trace, l->data);
 			trace = g_list_append (trace, NULL);
+			trace = g_list_append (trace, NULL);
```

Comment:
whar's this 3rd value?

Reply:
/* Only for interpreter frames */
	MonoJitInfo *ji;
}  ExceptionTraceIp;

Reply:
Its used by ves_icall_get_trace ().